<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="telas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Closet</string>
     </attribute>
     <widget class="QLabel" name="teste">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Novo</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>411</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nome: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputNome"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Tipo de peça: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="inputTipo">
         <item>
          <property name="text">
           <string>Camisa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Calça</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Calçado</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bermuda</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Shortinho</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Cor: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="inputCor">
         <item>
          <property name="text">
           <string>Preto</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Azul</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Branco</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vermelho</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Tamanho: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputTamanho"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnCadastro">
         <property name="text">
          <string>Cadastrar nova peça</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tabela">
         <column>
          <property name="text">
           <string>Nome</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tipo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Cor</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tamanho</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>370</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Ordenar por:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="btnOrdenar">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nome</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tipo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cor</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tamanho</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Match</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Historico</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Estatisticas</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>475</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Sobre</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTeste">
    <property name="title">
     <string>Teste</string>
    </property>
    <addaction name="actionAdicionar"/>
   </widget>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar"/>
   </widget>
   <addaction name="menuSobre"/>
   <addaction name="menuTeste"/>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdicionar">
   <property name="text">
    <string>Adicionar</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
